setup.py

```diff
@@ -14,8 +19,10 @@
             "tests",
         ]
     ),
+    include_package_data=True,
     install_requires=[
         "attrs>=21.3.0",
+        "itemadapter>=0.2.0",
```

Comment:
Is this OK? I think it may have been a mistake on my part not to include it when adding the adapter.

Reply:
Note to self: remove from `tox.ini` if this is OK.

Reply:
yes, it should be a dependency, nice catch @Gallaecio!